docs/Makefile

```diff
@@ -39,10 +39,15 @@ attributes:
 		> "${java_managed_partials}/attributes.adoc"
 	echo ":java-version: 11" \
 		>> "${java_managed_partials}/attributes.adoc"
+	echo ":spring-java-version: 17" \
+		>> "${java_managed_partials}/attributes.adoc"
+	echo ":minimum_docker_version: 19.03" \
+		>> "${java_managed_partials}/attributes.adoc"
```

Comment:
I don't know why, but this was not being added to the attributes, but load each time by the include.adoc. 

Anyway, better to manage the versions from a single place.

Reply:
Btw, I don't even think we recommend docker 19.03 all over the places. I think I have seen other versions in other places. Probably hardcoded in the adoc files.

Reply:
I wasn't aware of this